tools/vms-generator/utils/utils.go

```diff
@@ -699,7 +699,7 @@ func getBaseVM(name string, labels map[string]string) *v1.VirtualMachine {
 			Labels: labels,
 		},
 		Spec: v1.VirtualMachineSpec{
-			Running: &running,
+			RunStrategy: &runStrategy,
```

Comment:
Can we use the pointer kubevirt package here?
```suggestion
			RunStrategy: pointer.P(v1.RunStrategyHalted),
```